js/apps/admin-ui/src/components/users/UserDataTableToolbarItems.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  Button,
+  ButtonVariant,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+  ToolbarItem,
+} from "@patternfly/react-core";
+import { useAccess } from "../../context/access/Access";
+
+import type RealmRepresentation from "@keycloak/keycloak-admin-client/lib/defs/realmRepresentation";
+
+type UserDataTableToolbarItemsProps = {
+  realm: RealmRepresentation;
+  hasSelectedRows: boolean;
+  toggleDeleteDialog: any;
+  toggleUnlockUsersDialog: any;
+  goToCreate: any;
```

Comment:
We don't want to use `any` as this defeats the purpose of using typescript

Reply:
Fixed